### ISSUE TITLE
对qwen2-7b  model 进行sft loss 异常 

### ISSUE THREAD

**Issue by wccccp** [modelscope/ms-swift#2516]
**Describe the bug**
What the bug is, and how to reproduce, better with screenshots(描述bug以及复现过程，最好有截图)
![image](https://github.com/user-attachments/assets/25c298ac-f52d-4d41-a7ea-0480826cec9e)

**Your hardware and system info**
Write your system info like CUDA version/system/GPU/torch version here(在这里给出硬件信息和系统信息，如CUDA版本，系统，GPU型号和torch版本等)
4090*8 
环境是官方的v0.2.0 docker 

**Additional context**
Add any other context about the problem here(在这里补充其他信息)
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \
swift sft \
    --model_type qwen2-7b \
    --model_revision master \
    --sft_type full \
    --ddp_backend nccl \
    --template_type AUTO \
    --dtype AUTO \
    --output_dir output \
    --ddp_backend nccl \
    --dataset belle-train-1M-CN \
    --train_dataset_sample -1 \
    --num_train_epochs 3 \
    --max_length 2048 \
    --check_dataset_strategy warning \
    --gradient_checkpointing true \
    --batch_size 1 \
    --weight_decay 0.1 \
    --learning_rate 5e-5 \
    --gradient_accumulation_steps 32 \
    --max_grad_norm 0.5 \
    --warmup_ratio 0.03 \
    --eval_steps 100 \
    --save_steps 100 \
    --save_total_limit 2 \
    --logging_steps 1 \
    --use_flash_attn true \
    --lazy_tokenize true \
    --deepspeed 'zero3-offload' \
    --save_only_model true \
    --report_to wandb

**Comment by Jintao-Huang** [comment]
--template_type 请设置为 qwen

**Comment by Jintao-Huang** [comment]
全参数训练 学习率请下降到3e-6 - 1e-5
